package.json

```diff
@@ -15,6 +15,7 @@
     "format:fix": "pretty-quick --staged",
     "format:all": "npx prettier --write \"src/**/*{.ts,.js,.json,.css,.scss}\"",
     "precommit": "run-s format:fix lint gitSecrets scanRepo",
```

Comment:
Maybe we should take gitSecrets and scanRepo out of precommit too?

Reply:
hym, after our discussion that we had yesterday I agree with @ataylorBJSS  :)